components/Footer.vue

```diff
@@ -6,41 +6,114 @@
   <footer class="footer">
     <MaxWidthContainer class="footer-inner">
       <div class="footer-socials">
-        <img src="~/assets/images/TopBarLogo.svg" alt="Redot logo" class="footer-logo" />
+        <img
+          src="~/assets/images/TopBarLogo.svg"
+          alt="Redot logo"
+          class="footer-logo"
+        >
         <div class="footer-spacer" />
-        <img src="~/assets/images/social-github.svg" alt="GitHub" class="footer-social" />
-        <img src="~/assets/images/social-discord.svg" alt="Discord" class="footer-social" />
-        <img src="~/assets/images/social-twitter.svg" alt="Twitter" class="footer-social" />
-        <img src="~/assets/images/social-reddit.svg" alt="Reddit" class="footer-social" />
+        <img
+          src="~/assets/images/social-github.svg"
+          alt="GitHub"
+          class="footer-social"
+        >
+        <img
+          src="~/assets/images/social-discord.svg"
+          alt="Discord"
+          class="footer-social"
+        >
+        <img
+          src="~/assets/images/social-twitter.svg"
+          alt="Twitter"
+          class="footer-social"
+        >
+        <img
+          src="~/assets/images/social-reddit.svg"
+          alt="Reddit"
+          class="footer-social"
+        >
       </div>
 
       <div class="footer-columns">
         <div class="footer-column">
-          <SectionTitle small>Redot Engine</SectionTitle>
-
-          <NuxtLink href="#" class="footer-link">Download</NuxtLink>
-          <NuxtLink href="#" class="footer-link">Documentation</NuxtLink>
-          <NuxtLink href="#" class="footer-link">Source Code</NuxtLink>
+          <SectionTitle small>
+            Redot Engine
+          </SectionTitle>
```

Comment:
personally i like when its compact if the everything could be inlined.

Reply:
Did a little digging around. Looks like it's caused by this:

https://eslint.vuejs.org/rules/singleline-html-element-content-newline

I can choose to turn it off but it wouldn't auto format components that do need linebreaks

Reply:
yes, i think that is fine, we dont need linebreak when its not required.